scripts/release.sh

```diff
@@ -582,6 +700,28 @@ EOF
         --sort-by='.metadata.name' \
         -o custom-columns='Name:.metadata.name' --no-headers)
 
+    ALL_ARR_1=$(echo "${ALL_NODES}" | cut -d $'\n' -f1-250)
+    ALL_ARR_2=$(echo "${ALL_NODES}" | cut -d $'\n' -f251-500)
+    ALL_ARR_3=$(echo "${ALL_NODES}" | cut -d $'\n' -f501-750)
+    ALL_ARR_4=$(echo "${ALL_NODES}" | cut -d $'\n' -f751-1000)
+    ALL_ARR_5=$(echo "${ALL_NODES}" | cut -d $'\n' -f1001-1250)
+    ALL_ARR_6=$(echo "${ALL_NODES}" | cut -d $'\n' -f1251-1500)
+    ALL_ARR_7=$(echo "${ALL_NODES}" | cut -d $'\n' -f1501-1750)
+    ALL_ARR_8=$(echo "${ALL_NODES}" | cut -d $'\n' -f1751-2000)
+    ALL_ARR_9=$(echo "${ALL_NODES}" | cut -d $'\n' -f2001-2250)
+    ALL_ARR_10=$(echo "${ALL_NODES}" | cut -d $'\n' -f2251-2500)
```

Comment:
may be to safe add 2501-2750 and 2751-3000. Just in case we have more nodes.

Reply:
ok